src/Sushi.Mediakiwi/Controllers/Supporting/BaseController.cs

```diff
@@ -1,41 +1,41 @@
 ﻿using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
+using Sushi.Mediakiwi.Data;
 using System.IO;
 using System.Text.Json;
 using System.Threading.Tasks;
 
 namespace Sushi.Mediakiwi.Controllers
 {
-    public class BaseController : ControllerBase, IController
+    public class MediakiwiController : ControllerBase, IController
```

Comment:
File is still named BaseController.cs, please resolve

Reply:
resolved.